documentation/tools/oracle-factory/quick-start.md

```diff
@@ -1,174 +1,88 @@
 
-const readOracleRes = await factory.readOracle(createOracleRes.cid); // content ID C id
-console.log("readOracleRes: ", readOracleRes);
-```
+### **1.4. Instantiate SDK**
 
-### **3.2. Create, Update, and Read Oracle**
+Import and initialize the Oracle Factory SDK in your application.
 
-Utilize the SDK's methods to create, update, and read from the oracle.
+{% tabs %}
+{% tab title="Browser" %}
+Instantiate the SDK in your front-end project:
 
-#### **3.2.1. Creating an Oracle**
+#### 2.4.1. Basic Instantiation
 
 ```javascript
-// createOracle.js
-const createOracle = async (factory, params) => {
-  try {
-    const observable = new Promise((resolve, reject) => {
-      let paramSet;
-      let cid;
-
-      factory.createOracle(params).subscribe({
-        error: (e) => reject(e),
-        next: (value) => {
-          const { message, ...additionalEntries } = value;
-          if (message === "PARAM_SET_CREATED") {
-            paramSet = additionalEntries.paramSet;
-          }
-          if (message === "PARAM_SET_UPLOADED") {
-            cid = additionalEntries.cid;
-          }
-          console.log(message);
-          console.info(JSON.stringify(additionalEntries));
-        },
-        complete: () => {
-          console.log(`Oracle created, paramSet CID is ${cid}!`);
-          console.log(`paramSet: "${paramSet}"`);
-          resolve({ paramSet, cid });
-        },
-      });
-    });
-    return await observable;
-  } catch (error) {
-    throw error;
-  }
-};
-
-export default createOracle;
+import { IExecOracleFactory } from "@iexec/iexec-oracle-factory-wrapper";
 
+const web3Provider = window.ethereum;
+// instantiate
+const factory = new IExecOracleFactory(web3Provider);
 ```
+{% endtab %}
 
-#### **3.2.2. Updating an Oracle**
-
-```javascript
-// updateOracle.js
-const updateOraclePromise = async (factory, params) => {
-  let resolveValue;
-  try {
-    const observable = new Promise((resolve, reject) => {
-      factory
-        .updateOracle(params, {
-          workerpool: "0x0e7bc972c99187c191a17f3cae4a2711a4188c3f", // TODO: specifier un WP par defaut (required)
-          targetBlockchains: ["134", "137"], // TODO add this to documentation + BC supporté par OF (ross - chain) (optional - default 134 = bellecour)
-        })
-        .subscribe({
-          error: (e) => reject(e),
-          next: (value) => {
-            const { message, ...rest } = value;
-            console.log(message);
-            if (message === "TASK_UPDATED") {
-              resolveValue = rest;
-            }
-          },
-          complete: () => {
-            resolve(resolveValue);
-          },
-        });
-    });
-    return await observable;
-  } catch (error) {
-    throw error;
-  }
-};
-
-export default updateOraclePromise;
-```
+{% tab title="NodeJS" %}
+Instantiate the SDK in your back-end project:
 
-#### **3.2.3. Reading from an Oracle**
+#### 2.4.1. Basic Instantiation
```

Comment:
```suggestion
#### 1.4.1. Basic Instantiation
```

Reply:
addressed in https://github.com/iExecBlockchainComputing/documentation-tools/pull/27